analysis/connection.R

```diff
@@ -0,0 +1,45 @@
+# author: Daniel Câmara
+# date: March 3rd, 2022
+
+system2("ssh", c("epigraph@epigraphhub.org -L 5432:localhost:5432", "-NC"), wait = FALSE)
+
+choice_env <- menu(title = "Would you like to load your .env file?", 
```

Comment:
This should be removed because if the user wants to create the connection, `yes` is the only possible answer to this question.

Reply:
I can't understand why this should be removed. The system2() function is used to run the connection through the terminal. If it is removed, it is not possible to connect to the server.

Reply:
Please, see how this is done now in the new push.